src/Uno.UI/UI/KnownColors.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// MUX Reference KnownColors.h, tag winui3/release/1.6-stable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.UI;
+
+internal enum KnownColors : uint
```

Comment:
Is this used?

Reply:
Not at the moment, no, but could be useful in the future, it is synced with the WinUI source. Should I remove for now?

Reply:
Maybe keep it with `#if false` so it's easily restorable when needed, not sure. @MartinZikmund thoughts?

Reply:
Agreed with Martin, I will keep it with #if false